pages/landing/hipaa/template/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Achieve HIPAA compliant infrastructure, instantly
+excerpt: Go live with a customized, HIPAA-compliant cloud infrastructure and sample application in about a day.
```

Comment:
1. Makes sense 
2. Instantly is probably wrong - about a day is more appropriate most likely 
3. We believe that targeting _both_ HIPAA-compliant infrastructure and apps (by way of providing one scaffold perhaps in node.js to start - and, eventually, many others in the most popular languages) is both achievable and desirable. 
4. This was something that came out of our brainstorming discussions - our initial read is that while the HIPAA use case is likely to require the exact same ongoing maintenance as a non-HIPAA-compliant ref arch, it is unlikely that the compliance requirements for HIPAA/HITECH are going to change significantly or with any appreciable frequency. This makes it a little trickier to message around the subscription itself. 
5. It appears as though HIPAA is used interchangeably with HIPAA/HITECH, however some web properties do include HITECH as well. It might benefit us from an SEO perspective to include both on our page - but given that HIPAA provided the underlying framework and HITECH was mostly centered around strengthening enforcement options for breaches of the HIPAA standard, it seems we could safely elide the HITECH acronym in most of our copy. See also: https://www.connectria.com/blog/difference-in-hipaa-hitech/

Reply:
Fixed in ec6b8e56

Reply:
> it is unlikely that the compliance requirements for HIPAA/HITECH are going to change significantly or with any appreciable frequency

The high-level requirements for HIPAA/HITECH won't change, but the tools & techniques you use to implement them will change! Terraform, AWS, Packer, Kubernetes, and the entire DevOps industry are _constantly_ changing, as are auditor expectations, and what may have been HIPAA compliant best practices 18 months ago probably isn't today. We should communicate that clearly.


> It might benefit us from an SEO perspective to include both on our page - but given that HIPAA provided the underlying framework and HITECH was mostly centered around strengthening enforcement options for breaches of the HIPAA standard, it seems we could safely elide the HITECH acronym in most of our copy.

Roger.

Reply:
Expanded messaging in 10a5bd7e to include the fact that tools, best practices and auditor expectations are constantly changing.